style.css

```diff
@@ -0,0 +1,918 @@
+.backimg {
+  background: url(./img/Background-img-mobile.png) no-repeat center center fixed;
+  background-size: cover;
+  height: 100vh;
+  padding-bottom: 11vh;
+  margin-left: -7vw;
+  margin-right: -7vw;
+  margin-top: -3.5vh;
+}
+
+.backimg-about {
+  background: url(./img/about-desktop-back.png) no-repeat center center fixed;
+  background-size: cover;
+  height: 100vh;
+  margin-left: -7vw;
+  margin-right: -7vw;
+  padding-bottom: 11vh;
+  margin-top: -3.5vh;
```

Comment:
[Optional]
- [ ] Negative margins have incorrect behavior on mobile devices and affect elements that occur after them in the document flow. You should avoid using them.